src/main/java/uk/gov/ch/model/update/trusts/CorporateTrusteeData.java

```diff
@@ -0,0 +1,169 @@
+package uk.gov.ch.model.update.trusts;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+
+@Entity
+public class CorporateTrusteeData {
+
+    @Column(name = "TRUSTEE_ID")
+    @JsonProperty("trusteeId")
+    private Long trusteeId;
+
+    @Column(name = "TRUSTEE_NAME")
+    @JsonProperty("trusteeName")
+    private String trusteeName;
+
+    @Column(name = "REGISTER_LOCATION")
+    @JsonProperty("registerLocation")
+    private String registerLocation;
+
+    @Column(name = "REGISTRATION_NUMBER")
+    @JsonProperty("registrationNumber")
+    private String registrationNumber;
+
```

Comment:
@Column(name="country")
@JsonProperty("country")
private String country;

Reply:
yes please - must have missed it. Maybe mistaken for address.

Reply:
Done